src/kudu/consensus/consensus_meta.h

```diff
@@ -43,6 +43,15 @@ enum class ConsensusMetadataCreateMode {
   NO_FLUSH_ON_CREATE,
 };
 
+// Last known leader information.
+struct LastKnownLeader {
```

Comment:
why are we not reusing PreviousVotePB? The structs seem to be similar.

Reply:
I thought about it but the context was completely different so I chose this to make things really clear. LMK what you think.

Reply:
Something that I think is weird about putting this into ConsensusMeta is I don't see anywhere this is being persisted. ConsensusMeta is full of mostly persistent metadata. Any plans to persist this? Also do we need the chain or just the last one.